packages/editor/src/client/routes/imageList.tsx

```diff
@@ -32,36 +32,44 @@ import {
   Icon,
   Input,
   InputGroup,
-  Panel,
   IconButton,
   Drawer,
   Modal,
-  Button
+  Button,
+  Table
 } from 'rsuite'
-import {Overlay} from '../atoms/overlay'
-import {Typography} from '../atoms/typography'
 
-const ImagesPerPage = 24
+import {DEFAULT_TABLE_IMAGE_PAGE_SIZES} from '../utility'
+
+const {Column, HeaderCell, Cell, Pagination} = Table
 
 export function ImageList() {
   const {current} = useRoute()
   const dispatch = useRouteDispatch()
+  const [images, setImages] = useState<FullImageFragment[]>([])
+
+  const [filter, setFilter] = useState('')
+
+  const [isConfirmationDialogOpen, setConfirmationDialogOpen] = useState(false)
+  const [currentImage, setCurrentImage] = useState<ImageRefFragment>()
+
+  const [activePage, setActivePage] = React.useState(1)
```

Comment:
Its in the imports so you shouldn't need the `React.` here I think.

Reply:
Good point thanks